server/src/main/scala/actors/FlightMessageConversion.scala

```diff
@@ -41,28 +41,28 @@ object FlightMessageConversion {
 
   def apiFlightToFlightMessage(apiFlight: Arrival): FlightMessage = {
     FlightMessage(
-      operator = Some(apiFlight.Operator),
-      gate = Some(apiFlight.Gate),
-      stand = Some(apiFlight.Stand),
+      operator = apiFlight.Operator,
```

Comment:
we should make absolutely certain we are saving and restoring these things correctly.